src/core_plugins/kibana/public/home/components/home.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Synopsis } from './synopsis';
+import {
+  KuiButton,
+  KuiFlexGroup,
+  KuiFlexItem,
+  KuiFlexGrid,
+} from 'ui_framework/components';
+import { FeatureCatalogueCategory } from 'ui/registry/feature_catalogue';
+
+export function Home({ addBasePath, directories }) {
+
+  const renderDirectories = (category) => {
+    return directories
+    .filter((directory) => {
+      return directory.showOnHomePage && directory.category === category;
+    })
+    .map((directory) => {
+      return (
+        <KuiFlexItem style={{ minHeight: 64 }} key={directory.id}>
+          <Synopsis
+            description={directory.description}
+            iconUrl={addBasePath(directory.icon)}
+            title={directory.title}
+            url={addBasePath(directory.path)}
+          />
+        </KuiFlexItem>
+      );
+    });
+  };
+
+
+  return (
+    <div className="kuiView home">
+      <div className="kuiViewContent">
+
+        <div className="kuiViewContentItem kuiVerticalRhythmXLarge">
+          <KuiFlexGroup
+            className="kuiVerticalRhythmSmall"
+            justifyContent="spaceBetween"
+            alignItems="flexEnd"
+          >
+            <KuiFlexItem>
+              <h1 className="kuiTitle">
+                Welcome to Kibana
+              </h1>
+            </KuiFlexItem>
+
+            <KuiFlexItem grow={false}>
+              <KuiFlexGroup alignItems="center">
+                <KuiFlexItem grow={false}>
+                  <p className="kuiText kuiSubduedText">
+                    Data already in Elasticsearch?
+                  </p>
+                </KuiFlexItem>
+
+                <KuiFlexItem grow={false}>
+                  <a href={addBasePath('/app/kibana#/management/kibana/index')}>
+                    <KuiButton buttonType="secondary">
```

Comment:
This will render a `<button>` inside an `<a>`, which is not quite the desired result (e.g. for accessibility reasons, but I could imagine some other stuff might also be broken (or close to it) with that).

You should apply the CSS classes directly:

```html
<a href=".." class="kuiButton kuiButton--secondary">
  <span class="kuiButton__inner">
    Set up index pattern
  </span>
</a>
```